tools/devops/automation/templates/devices/build.yml

```diff
@@ -55,14 +55,6 @@ steps:
     BUILD_REVISION: 'jenkins'
   continueOnError: true
 
-- bash: |
-    security set-key-partition-list -S apple-tool:,apple: -s -k $KEYCHAIN_PASS login.keychain
-  env:
-    KEYCHAIN_PASS: ${{ parameters.keyringPass }}
-  displayName: 'Remove security UI-prompt (http://stackoverflow.com/a/40039594/183422)'
-  condition: succeededOrFailed() # we do not care about the previous process cleanup
-  continueOnError: true
-
```

Comment:
We don't longer need it?

Reply:
correct. you will still want to set it on xamarin-macios.keychain, but the login.keychain should no longer be used in CI.